pkg/acsengine/engine.go

```diff
@@ -1746,7 +1746,7 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) templat
 			return helpers.IsTrueBoolPointer(cs.Properties.OrchestratorProfile.KubernetesConfig.EnablePodSecurityPolicy)
 		},
 		"OpenShiftGetMasterSh": func() string {
-			tb := MustAsset("k8s/openshiftmasterscript.sh")
+			tb := MustAsset("openshift/openshiftmasterscript.sh")
```

Comment:
use `openshiftMasterScript` here

Reply:
Let's stick with lowercased letters since that's what's followed elsewhere?

Reply:
sorry, unclear review from me - I mean, use the constant called `openshiftMasterScript` which is already defined